include/notincluded/polytopes-NOTINC.h

```diff
@@ -0,0 +1,638 @@
+// VolEsti
+
+// Copyright (c) 2012-2017 Vissarion Fisikopoulos
+
+// VolEsti is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or (at
+// your option) any later version.
+//
+// VolEsti is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+// for more details.
+//
+// See the file COPYING.LESSER for the text of the GNU Lesser General
+// Public License.  If you did not receive this file along with HeaDDaCHe,
+// see <http://www.gnu.org/licenses/>.
+
+#ifndef POLYTOPES_H
+#define POLYTOPES_H
+
+#define CGAL_QP_NO_ASSERTIONS
+
+//this is for LP-solver
+#include <iostream>
+//#include <CGAL/basic.h>
+//#include <CGAL/QP_models.h>
+//#include <CGAL/QP_functions.h>
+// choose exact integral type
+//#ifdef CGAL_USE_GMP
+//#include <CGAL/Gmpzf.h>
+//typedef CGAL::Gmpzf ET;
+//#endif
+//typedef double ET;
+//#else
+//#include <CGAL/MP_Float.h>
+//typedef CGAL::MP_Float ET;
+//#endif
+//#include <boost/random/shuffle_order.hpp>
+#include "rref.h"
+//#include "LPsolve/solve_lp.h"
+//EXPERIMENTAL
+//to implement boundary oracles using NN queries  
+//#include <flann/flann.hpp>
+
+
+// ellipsoid class
+template <typename K>
```

Comment:
is this duplicated?

Reply:
Yes everything in folder notincluded it is duplicated and not included. I am going to remove this folder.